### DISCUSSION TITLE
LastOnline Suggestion

### DISCUSSION THREAD

**Discussion by BboyZeroOne** [Zrips/CMI#8066] [Feature Requests]
I would like to suggest a lastonline to show/display a player's last online in time or day.

**Placeholders:**
%cmi_user_lastonline% - Checks player's last online time/day.
Usage scenario:
Player was last online since %cmi_user_lastonline%.  =   Player was last online since 3days 15minutes.

**Specialized Commands:**
lastonline:[value][@][?][#]! - Checks player's last online time. Usage scenario:
_lastonline:30d! cmi titlemsg [playerName] Welcome back! You were offline for 30 days!;;asConsole! cmi money give [playerName] 100_

**Comment by CloudeLecaw** [comment]
You got:
%player_last_join% %player_last_played_formatted% %player_last_join_date%
well at least with theese you could do something like a check: and cmis maths placeholder
  might just need to look that the formats matches for the calculation.
it could look like this...
`%cmi_equationint_{server_time_MM:dd:yyyy}-{player_last_join_date}%>30! cmi titlemsg [playerName] Welcome back! You were offline for more than 30 days.`

**Reply by BboyZeroOne** [reply]
It would be nice if we could do away with the cmi's equation. With a dedicated and simplified last-online cmi placeholder, it would also be beneficial with the usage of specialized commands. It would also specifically mention how many days a player has been offline for.

**Reply by CloudeLecaw** [reply]
i agree i just ment its currently possible if really bad wanted ;D

**Reply by BboyZeroOne** [reply]
yep, hopefully they'll add this in! <3
